package.json

```diff
@@ -109,16 +109,15 @@
     "vue-jest": "^3.0.7"
   },
   "engines": {
-    "node": ">= 12.0.0",
-    "npm": ">= 6.0.0"
+    "node": "16.13.0",
+    "npm": "8.1.0"
```

Comment:
Should we be pinning `engines` this aggressively? I liked the old approach that only pinned the major version.